config/routes.rb

```diff
@@ -21,6 +21,12 @@
 
     resources :locations, only: [:index, :show]
 
+    resources :appointment_summaries, only: %i(new create), path: 'summary-document' do
+      post :view, on: :collection
+      post :download, on: :collection
+      post :print, on: :collection
+    end
```

Comment:
Would the below be nicer - allows us tot use show instead of view - using resource instead of resources?

```
resource :appointment_summaries, only: %i(new create show), path: 'summary-document' do
  post :download, on: :member
  post :print, on: :member
end
```